### ISSUE TITLE
Flaky tests: TransportThatFallsbackCreatesNewConnection et al

### ISSUE THREAD

**Issue by ryanbrandenburg** [aspnet/SignalR#1629]
[This](http://aspnetci/viewLog.html?buildId=423387&buildTypeId=XPlat_Windows_Win2008r2_Universe) test and others sometimes fails with a message like:
```
System.TimeoutException : Operation in TransportThatFallsbackCreatesNewConnection timed out at C:\BuildAgent\work\33bdfc1cae7b2a38\modules\SignalR\test\Microsoft.AspNetCore.SignalR.Tests\EndToEndTests.cs:69
   at System.Threading.Tasks.TaskExtensions.OrTimeout(Task task, TimeSpan timeout, String memberName, String filePath, Nullable`1 lineNumber)
   at Microsoft.AspNetCore.SignalR.Tests.EndToEndTests.TransportThatFallsbackCreatesNewConnection() in C:\BuildAgent\work\33bdfc1cae7b2a38\modules\SignalR\test\Microsoft.AspNetCore.SignalR.Tests\EndToEndTests.cs:line 69
--- End of stack trace from previous location where exception was thrown ---
```

This particular subset of failures is being tracked in a larger list [here](http://aspnetci/httpAuth/app/rest/builds?locator=tag:timeout,sinceDate:201710010T000000-0001&fields=build(startDate,buildTypeId,status,branchName,webUrl,tags)).

**Comment by ryanbrandenburg** [comment]
[This](http://aspnetci/project.html?tab=testDetails&testNameId=-1869633753029778556&projectId=XPlat&buildTypeId=XPlat_HighSierraOsxUniverse) fails more often than it passes on High Sierra.

**Comment by BrennanConroy** [comment]
@mikaelm12

**Comment by davidfowl** [comment]
I saw this locally as well.

**Comment by mikaelm12** [comment]
This change should fix this: 81b0f60b5ae7a3efa2caab91a655d0099b4f2da2
Closing for now. We can reopen if the problem persists.
